src/strongbox/addon.clj

```diff
@@ -29,15 +29,30 @@
 
 (defn-spec remove-addon nil?
   "removes the given addon. if addon is part of a group, all addons in group are removed"
-  [addon-dir ::sp/addon-dir, toc :addon/toc]
-  (if (contains? toc :group-addons)
-    ;; top-level toc is contained in the :group-addons list
-    (doseq [subtoc (:group-addons toc)]
-      (-remove-addon addon-dir (:dirname subtoc)))
-    (-remove-addon addon-dir (:dirname toc))))
+  [addon-dir ::sp/addon-dir, addon :addon/installed]
```

Comment:
review usage of `addon/installed` - is it more or less permissive than `toc` ? if less permissive, then revert.